go/ql/src/InconsistentCode/DeferredErrorCall.ql

```diff
@@ -0,0 +1,22 @@
+/**
+ * @name Deferred call may return an error
+ * @description Deferring a call to a function which may return an error means that an error may not be handled.
+ * @kind problem
+ * @problem.severity warning
+ * @id go/deferred-error-call
+ * @tags maintainability
+ *  correctness
+ *  call
+ *  defer
+ */
+
+import go
+
+from DeferStmt defer, SignatureType sig
+where
+  // match all deferred function calls and obtain their type signatures
+  sig = defer.getCall().getCalleeExpr().getType().(SignatureType) and
```

Comment:
```suggestion
  sig = defer.getCall().getCalleeExpr().getType() and
```
(I see ql-for-ql has also picked this up.)

Reply:
Fixed in [c907a9e](https://github.com/github/codeql/pull/11410/commits/c907a9e70719cd4521a43275b5585ddd5751eae0)